frontend/src/pages/product-search/ProductSearch.jsx

```diff
@@ -127,64 +127,77 @@ function ProductSearch() {
 
-                <Navbar isComparisonNav={false} />
-
-                <h1 className="center-text max-width-heading">Compare products</h1>
-                <br/>
-                <div className="search-and-selected">
-                    <p className="body-1">
-                        <span className="body-1-bold">Select 3 or more</span> products to compare predicted likability and details
-                    </p>
-                    <p className="body-1-medium num-selected min-selection">
-                        {numProductsSelected} Selected
-                    </p>
-                </div>
-                <SearchBar
-                    isButtonInsideBar={true}
-                    onSearchSubmit={onSearchSubmit}
-                    query={searchQuery}
-                    setQuery={setSearchQuery}
-                />
+                <Navbar/>
 
-                {showRecommendations && !isPreferencesSkipped && <div>
-                    <br/>
-                    <h4>Picked for you</h4>
-                    <p className="body-1 max-width-body">Based on how much you liked previous recommendation rankings and answers to question</p>
+                <Link className="body-1-medium back" to={routes.home}>← Back</Link>
+
+                <div className="content-product-search">
+                    <h1 className="center-text max-width-heading">Compare products</h1>
                     <br/>
-                    <div className="picked-for-you">
-                        {productData.slice(0, numPickedForYou).map(product => (
-                            <ProductOption
-                                data={product}
-                                selectionNumber={productSelectionNumber(product)}
-                                changeSelection={onProductSelection}
-                            />
-                        ))}
+                    <div className="search-and-selected">
+                        <p className="body-1">
+                            <span className="body-1-bold">Select 3 or more</span> products to compare predicted likability and details
+                        </p>
+                        <p className="body-1-medium num-selected min-selection">
+                            {numProductsSelected} Selected
+                        </p>
                     </div>
+                    <SearchBar
+                        isButtonInsideBar={true}
+                        onSearchSubmit={onSearchSubmit}
+                        query={searchQuery}
+                        setQuery={setSearchQuery}
+                    />
                     <br/>
-                    <br/>
-                </div>}
 
-                <div>
+                    {productData.length == 0 && <div className="all-products-empty">
+                        <h4>Search products</h4>
+                        <p className="body-1">Search for an item, or enter the specific URL of the products you want to compare</p>
+                    </div>}
```

Comment:
I definitely agree that we should have a section like this if nothing has been searched yet, but let's not forget to add a Figma design for it since it is a pretty substantial change/addition.

Reply:
Yes for sure!